recipes/seadex-essentials/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.59"
+
+class SeadexEssentialsConan(ConanFile):
+    name = "seadex-essentials"
+    description = "essentials is a small c++ library that offers very basic capabilities for applications and libraries."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://essentials.seadex.de/"
+    topics = ("utility", "C++", "library")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "spdlog/*:header_only": True
```

Comment:
Only if it's necessary to build this library, should this option be set for `spdlog`, and if that's the case you need to instead override it in the `configure` method and additionally need to add a check to the `validate` method. This is documented [here](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#handling-requirements-options). 

```suggestion
```

Reply:
Thanks for the suggestion. I want to do it like this, but I tried to add the line:

`self.options["spdlog/*"].header_only = True`

in the _configure()_ method but it fails to build then. And I don't get why.

The log can be found here:
https://c3i.jfrog.io/c3i/misc/logs/pr/16654/72-linux-gcc/seadex-essentials/2.1.2//f0161b409d880731878faa07ad0fef999c33f11e-build.txt

Reply:
This is a bug and should not work either way https://github.com/conan-io/conan-center-index/pull/17000 will break this

You should update you code to check `SPDLOG_FMT_EXTERNAL` https://github.com/search?q=repo%3Agabime%2Fspdlog%20SPDLOG_FMT_EXTERNAL&type=code

Those bundled files should not be preset